LgiSystemServices/LgiSystemServices.cpp

```diff
@@ -212,5 +215,33 @@ namespace WPEFramework {
 
             returnResponse(true);
         }
+
+        uint32_t LgiSystemServices::getTimeZoneDST(const JsonObject& parameters,
+                JsonObject& response)
+        {
+            std::string timezone;
+            bool resp = false;
+
+            if (Utils::fileExists(TZ_FILE)) {
+                if(readFromFile(TZ_FILE, timezone)) {
+                    LOGWARN("Fetch TimeZone: %s\n", timezone.c_str());
+                    for(const char c : timezone) {
+                        LOGWARN("Fetch TimeZone: %c\n", c);
+                    }
+                    response["timeZone"] = timezone;
+                    response["accuracy"] = "FINAL";
+                    resp = true;
+                } else {
+                    LOGERR("Unable to open %s file.\n", TZ_FILE);
+                    response["timeZone"] = "null";
```

Comment:
shouldn't we 
populateResponseWithError(SysSrv_FileAccessFailed, response);
(just like below)?